lib/rollup/jobs/trends_job.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Castle.Rollup.Jobs.Trends do
+  # @redis Application.get_env(:castle, :redis)
```

Comment:
do you need this `@redis`?

Reply:
whoops!  i was actually looking at the comment in the other `totals_job.ex` ... which said I should try to use the public `@redis` interface instead of importing `Castle.Redis`.  But then i decided it was too hard to tackle right now.  Will replace with the comment!